akka-persistence-typed/src/main/resources/reference.conf

```diff
@@ -20,4 +20,25 @@ akka.persistence.typed {
   # this may happen while it receives commands while it is recovering events or while it is persisting events
   log-stashing = off
 
+  # Describes the algorithms and behavior of retention within
```

Comment:
From cluster sharding layer of the delete functions, prior to internal deletes. Additionally, as noted below, I have this idea flushed out in another branch to push for discussion/review after.